test/spoom/sorbet/sigils_test.rb

```diff
@@ -220,6 +220,26 @@ def test_change_sigil_in_file_with_iso_content
         project.destroy
       end
 
+      def test_change_sigil_in_file_with_default_internal_encoding
+        project = spoom_project
+
+        string = <<~RB
+          # typed: true
+
+          puts "À coûté 10€"
+        RB
+
+        old_encoding = Encoding.default_internal
+        Encoding.default_internal = Encoding::UTF_8
+
+        project.write("file.rb", string.encode("UTF-8"))
+        Sigils.change_sigil_in_file("#{project.path}/file.rb", "strict")
+        assert_equal("strict", Sigils.file_strictness("#{project.path}/file.rb"))
+        project.destroy
+
+        Encoding.default_internal = old_encoding
```

Comment:
Can we make this safer by putting it in an `ensure` block?
```suggestion
      ensure
        Encoding.default_internal = old_encoding if old_encoding
```
Otherwise, when/if the assertions above raise, the default internal encoding will not be restored, which might end up affecting other tests.